modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessor.java

```diff
@@ -147,11 +147,17 @@ protected DDMFormField getDDMFormField(JSONObject jsonObject) {
 	protected DDMFormFieldOptions getDDMFormFieldOptions(JSONArray jsonArray) {
 		DDMFormFieldOptions ddmFormFieldOptions = new DDMFormFieldOptions();
 
+		String defaultLocale = null;
+
 		for (int i = 0; i < jsonArray.length(); i++) {
 			JSONObject jsonObject = jsonArray.getJSONObject(i);
 
 			String value = jsonObject.getString("value");
 
+			if ((defaultLocale == null) && jsonObject.has("defaultLocale")) {
+				defaultLocale = jsonObject.getString("defaultLocale");
+			}
```

Comment:
`(defaultLocale == null)` will always be null because of line 150, so can we remove it from this condition?